_data/urvi367/urvi367.md

```diff
@@ -0,0 +1,6 @@
+---
+name: Urvi Rawat
+institution: Symbiosis Institute of Technology, Pune
+quote: The Office, Season 7, E19, 14:45. "Should have burned this place down when I had the chance."

```

Comment:
Hi @urvi367  we will need you to pick another quote to follow our code of conduct.

Reply:
Hi @juanpflores , I have made the requested changes.